src/modules/launcher/PowerLauncher/ResultList.xaml

```diff
@@ -59,7 +58,7 @@
         <ListView.ItemTemplate>
             <DataTemplate>
                 <Grid
```

Comment:
What about the ActualHeight of this, times the number of results?

Reply:
feels like it'd need some refactoring to make this work. I'll try it out as a follow-up PR